### ISSUE TITLE
ValueError when using `standardise_surface` with inlet value

### ISSUE THREAD

**Issue by rt17603** [openghg/openghg#477]
> For standardise.standardise_surface I'm now getting an error: ValueError: could not convert string to float: '185m'
I've put inlet = 185 so I think it must be changing it in the code to '185m' by default (we asked for it to do this for the other inputs so that it always uses 'm' instead of 'magl')

**Comment by hanchawn** [comment]
The code I ran to get this error:

obs_file = '/group/chemistry/acrg/obs/TAC/DECC-picarro_TAC_20130131_co2-185m-20220929.nc'

standardise.standardise_surface(filepaths=obs_file,
                                source_format='openghg',
                                site='TAC',
                                network='DECC,
                                inlet=185,
                                instrument='picarro',
                                sampling_period='1H',
                                overwrite=True)

**Comment by hanchawn** [comment]
Related to this, search_surface can't find results if an inlet is given.

This code:
search_results = search_surface(site='TAC',
                 species='co2',
                 inlet='185',
                 start_date='2018-01-01',
                 end_date='2018-02-01'
                 )

Gives this warning:
WARNING:openghg.retrieve:No data found for the dates given, please try a wider search.

Using the code above without a start_date and end_date does not return a warning but also does not return any results.

**Comment by rt17603** [comment]
I moved your update to a new issue as I think that'll be separate (but related).

**Comment by rt17603** [comment]
@hanchawn - quick question - if you run your standardise_surface line without passing an inlet at all, does this work?

**Comment by hanchawn** [comment]
Yep it works without an inlet

**Comment by rt17603** [comment]
Ok, so the error was coming in when we synchronised the attributes from the netcdf file and the created metadata. When an inlet value being passed this was checking against the attribute inlet_height_magl value of 185.0 within the netcdf file to make sure these were consistent but inlet_height_magl of "185m" could not be converted to a float. Updated code to fix this with extra check and improved definition.
